catalog/app/containers/Bucket/Meta.tsx

```diff
@@ -81,6 +81,7 @@ export function PackageMetaSection({ meta, preferences }: PackageMetaSectionProp
   const classes = usePackageMetaStyles()
   if (!meta || R.isEmpty(meta)) return null
   const { message, user_meta: userMeta, workflow } = meta
+  if (!message && !userMeta && !workflow) return null
```

Comment:
i think you should remove the `R.isEmpty(meta)` check above if you're adding this

Reply:
Oh, thanks